action.yml

```diff
@@ -65,6 +68,42 @@ inputs:
   git-ssh-known-host-keys:
     description: "Known SSH host keys for cloning private Git repositories via SSH (may be obtained using `ssh-keyscan`)"
 
+  # TODO: where to specify defaults? action.yml + docker-ros.yml vs ci.sh vs dockerfile
```

Comment:
TODO: where to specify defaults? action.yml + docker-ros.yml vs ci.sh vs dockerfile

Reply:
everywhere

Reply:
also keep defaults for additional special files